packages/nexus-sdk/src/Resource/types.ts

```diff
@@ -0,0 +1,29 @@
+import { Context, Resource } from '../types';
+
+export const DEFAULT_SCHEMA_ID = '_';
+
+export type ResourceList = {
```

Comment:
We already have a type for this, it should be 
```suggestion
export type ResourceList = PaginatedResources;
```